### PR TITLE
[FSPE-5202] Improved development workflow setup text

### PR DESCRIPTION
**Pull request by hanyjohny**

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4f446dc173a141c96b77d279c0b21377d655c44b-chi-pr.enterprise.ctl.io" target="_blank"> https://4f446dc173a141c96b77d279c0b21377d655c44b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1578/ for more details.

**Review by SwathiLingareddy — APPROVED** [review]
Changes are looking good for me

**Comment by hanyjohny** [comment]
Hi @SwathiLingareddy , please review this PR

File: src/website/views/getting-started/development-workflow.pug [hunk 1]
```diff
@@ -3,6 +3,9 @@ title: Development workflow
 order: 3
 ---
 h2 Setup
+p.-text
+  | GitHub repo to be forked and downloaded in order to enable the Dev Env workflow is 
+  a(href='https://github.com/CenturyLinkCloud/ux-chi') https://github.com/CenturyLinkCloud/ux-chi
```
Comment (jllr) [inline src/website/views/getting-started/development-workflow.pug:8]:
```suggestion
  | In order to set up Chi Development workflow, fork and clone 
  a(href='https://github.com/CenturyLinkCloud/ux-chi' target="_blank") ux-chi
  |  GitHub repository
  
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1605/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1607/ for more details.

**Comment by mattnickles** [comment]
test this please

**Review by jllr — APPROVED** [review]

**Comment by jllr** [comment]
test this please

**Comment by jllr** [comment]
test this please

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1609/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1612/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1613/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f33941aac19166ee505a9fe6aeeb38bbaf82886c-chi-pr.enterprise.ctl.io" target="_blank"> https://f33941aac19166ee505a9fe6aeeb38bbaf82886c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1614/ for more details.

File: src/website/views/getting-started/development-workflow.pug [hunk 2]
```diff
@@ -3,6 +3,11 @@ title: Development workflow
 order: 3
 ---
 h2 Setup
+p.-text
+  | In order to set up Chi Development workflow, fork and clone 
+  a(href='https://github.com/CenturyLinkCloud/ux-chi' target="_blank") ux-chi
+  | GitHub repository
```
Comment (jllr) [inline src/website/views/getting-started/development-workflow.pug:9]:
```suggestion
  |  GitHub repository
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://6c3f6c463ccf45d31e2fb0d85523693989c3e557-chi-pr.enterprise.ctl.io" target="_blank"> https://6c3f6c463ccf45d31e2fb0d85523693989c3e557-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1615/ for more details.

File: src/website/views/getting-started/development-workflow.pug [hunk 3]
```diff
@@ -3,6 +3,11 @@ title: Development workflow
 order: 3
 ---
 h2 Setup
+p.-text
+  | In order to set up Chi Development workflow, fork and clone 
+  a(href='https://github.com/CenturyLinkCloud/ux-chi' target="_blank") ux-chi
+  |  GitHub repository
```
Comment (jllr) [inline src/website/views/getting-started/development-workflow.pug:9]:
```suggestion
  |  GitHub repository.
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f62592c72a5057b43510b24aca58258db866307a-chi-pr.enterprise.ctl.io" target="_blank"> https://f62592c72a5057b43510b24aca58258db866307a-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1616/ for more details.

**Review by jllr — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-8pyis2, https://ops.elasticbox.com/services/instances/i-6i6u75, https://ops.elasticbox.com/services/instances/i-rbsql6, https://ops.elasticbox.com/services/instances/i-8gxtts

**Pull request merged**